src/Employer/Employer.Web/Views/Part2/Qualifications/Qualifications.cshtml

```diff
@@ -0,0 +1,205 @@
+﻿@using Esfa.Recruit.Employer.Web.ViewModels.Part2.Qualifications
+@using Esfa.Recruit.Vacancies.Client.Domain.Enums
+@model QualificationsViewModel
+
+
+<div class="grid-row">
+    <div class="column-full">
+
+        <h1 class="heading-xlarge small-bottom-margin">Qualifications</h1>
+
+        <div asp-validation-summary="All" class="error-summary" role="alert" aria-labelledby="error-summary-heading-example-1" tabindex="-1">
+            <h2 class="heading-medium error-summary-heading" id="error-summary-heading-example-1">
+                Please fix these errors
+            </h2>
+        </div>
+
+        <hr>
+
+        <form asp-route="@RouteNames.Qualifications_Post">
+
+            <div class="form-group form-group-compound">
+                <label asp-for="QualificationType" class="form-label-bold">Type of qualification</label>
+                <span asp-validation-for="QualificationType" class="error-message"></span>
+                <select asp-for="QualificationType" class="form-control sfa-form-control-auto">
+                    <option value="">Select from list</option>
+
+                    @foreach (var qualificationType in Model.QualificationTypes)
+                    {
+                        <option value="@qualificationType">@qualificationType</option>
+                    }
+                </select>
+            </div>
+
+            <div class="form-group form-group-compound">
+                <label asp-for="Subject" class="form-label-bold">Subject</label>
+                <span asp-validation-for="Subject" class="error-message"></span>
+                <input asp-for="Subject" type="text" class="form-control" maxlength="50">

```

Comment:
I think we decided just remove these sorts of `maxlength` attributes.

Reply:
@mgwilliam @leewadhams @sachpatel I know we've removed some of the HTML input validation attributes and pushing as much validation to VacancyClient as possible, but I think its worth keeping maxlength so users can't over type.

Reply:
@chrisjensenuk understood - can we expose a constant from the validation perhaps?

Reply:
@mgwilliam Good idea. We can do this when we revisit to add the validations.